Cargo.toml

```diff
@@ -15,9 +16,11 @@ edition = "2021"
 
 
 [workspace.dependencies]
-tokio = {version = "1", features = ["fs"]}
+tokio = { version = "1", features = ["fs"] }
+rust-eigenda-signers = { path = "./crates/rust-eigenda-signers" }
 tokio-stream = "0.1.16"
 rust-kzg-bn254 = "0.2.1"
+auto_impl = "1"
```

Comment:
any reason why this is done via a crate instead of adding the impls manually?

Reply:
No reason besides cutting down on boilerplate.

Do you wish me to inline these and get rid of the dep?

Reply:
yeah

Reply:
[4a5241f](https://github.com/segfault-magnet/eigenda-client-rs/commit/4a5241fe3c569170beea8670a2309861f910c640)